packages/ckeditor5-dev-utils/lib/translations/multiplelanguagetranslationservice.js

```diff
@@ -211,7 +211,16 @@ module.exports = class MultipleLanguageTranslationService extends EventEmitter {
 			const stringifiedTranslations = JSON.stringify( translatedStrings )
 				.replace( /"([a-z]+)":/g, '$1:' );
 
-			const outputBody = `CKEDITOR_TRANSLATIONS.add('${ language }',${ stringifiedTranslations })`;
+			const outputBody = (
+				// We need to ensure that the CKEDITOR_TRANSLATIONS variable exists and if it exists, we need to extend it.
+				// Use ES5 because this bit will not be transpiled!
+				'(function(d){' +
+					`d['${ language }']=Object.assign(` +
```

Comment:
It's still an ES6 code. We can change it to `for in` loop to be fully ES5 compatible.

Reply:
`Object.assign()` needs to be provided by a polyfill for the rest of the code anyway. This doesn't get transpiled.

Reply:
This might be tricky for someone, as we add that code at the top of the bundle.

Reply:
Right now we don't support any non-ES6 env anyway, so it's even hard to test. Let's keep it in our minds. But I don't see a problem in injecting the ES6 polyfill before our scripts.